extensionsGallery-insider.json

```diff
@@ -1784,6 +1784,612 @@
 					],
 					"statistics": [],
 					"flags": "preview"
+				},
+				{
+					"extensionId": "42",
+					"extensionName": "ads-language-pack-de",
+					"displayName": "German Language Pack for Azure Data Studio",
```

Comment:
So I assume that getting the gallery itself to be localized right now isn't something we're going to support? Have we thought about what that would take?

Reply:
No, that isn't in scope for what I'm doing here.